build/includes/sdk.mk

```diff
@@ -166,7 +166,7 @@ run-sdk-conformance-test-go:
 
 run-sdk-conformance-test-rust:
 	# run without feature flags
-	$(MAKE) run-sdk-conformance-test SDK_FOLDER=rust GRPC_PORT=9004 HTTP_PORT=9104
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=rust GRPC_PORT=9004 HTTP_PORT=9104 DELAY=1
```

Comment:
We'll need to add DELAY=1 to the rest of the invocations of `run-sdk-conformance-test` in this target.

Reply:
Done. Thanks!

Reply:
...and since we just merged #2444 we can drop this workaround, and things should work!

Reply:
Done. Thanks!